src/main/webui/src/app/components/content/common/StoreListingWidget.jsx

```diff
@@ -49,6 +49,42 @@ CapabilitiesSection.propTypes = {
   store: PropTypes.object
 };
 
+const ConstituentsSection = ({ store }) => {
+  const [toggle, setToggle] = useState(store.display === true);
+
+  store.display = toggle;
```

Comment:
Is this store.display = toggle neccessary? I think you can direct use toggle in following code instead of the store.display which is directly controlled by useState.

Reply:
removed it